core/domain/services/graphql/data/mutations/FormElementMutation.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace EventEspresso\core\domain\services\graphql\data\mutations;
+
+use GraphQLRelay\Relay;
+
+/**
+ * Class FormElementMutation
+ *
+ * @package       Event Espresso
+ * @author        Manzoor Wani
+ */
+class FormElementMutation
+{
+
+    /**
+     * Maps the GraphQL input to a format that the model functions can use
+     *
+     * @param array $input Data coming from the GraphQL mutation query input
+     * @return array
+     */
+    public static function prepareFields(array $input): array
+    {
+        $args = [];
+
+        if (isset($input['id'])) {
+            $args['FIN_UUID'] = sanitize_text_field($input['id']);
+        }
+
+        if (isset($input['belongsTo'])) {
+            $args['FSC_UUID'] = sanitize_text_field($input['belongsTo']);
+        }
+
+        if (isset($input['htmlClass'])) {
+            $args['FIN_htmlClass'] = sanitize_text_field($input['htmlClass']);
+        }
+
+        // order can be 0
+        if (array_key_exists('order', $input)) {
+            $args['FIN_order'] = absint($input['order']);
+        }
+
+        if (isset($input['status'])) {
+            $args['FIN_status'] = sanitize_text_field($input['status']);
+        }
+
+        if (! empty($input['wpUser'])) {
+            $parts = Relay::fromGlobalId(sanitize_text_field($input['wpUser']));
```

Comment:
isn't wpUser an int?

```suggestion
            $parts = Relay::fromGlobalId(absint($input['wpUser']));
```

Reply:
It's the GUID (string) that will be sanitized and passed to `Relay::fromGlobalId`